src/main/java/com/microsoft/graph/content/MSBatchRequestContent.java

```diff
@@ -0,0 +1,112 @@
+package com.microsoft.graph.content;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.http.Header;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpRequest;
+import org.apache.http.client.methods.HttpEntityEnclosingRequestBase;
+import org.apache.http.util.EntityUtils;
+import org.json.simple.JSONObject;
+import org.json.simple.JSONValue;
+
+public class MSBatchRequestContent {
```

Comment:
What is the reason for the MS prefix?  If a prefix is necessary, would "Graph" not be a better prefix?